Cargo.toml

```diff
@@ -35,6 +35,8 @@ version = "0.6"
 [dependencies.subtle]
 version = "^0.2"
 default-features = false
+git = "https://github.com/isislovecruft/subtle"
+branch = "feature/conditional-swap_r1"
```

Comment:
Assuming this is going to be swapped out when you tag a new subtle release

Reply:
I just tagged it like 10 minutes ago, so this is reverted in 49f87816